.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think these VS files are also here by mistake.

Reply:
Not sure why it did that